pages/index.tsx

```diff
@@ -32,8 +32,10 @@ const InputWrapper = styled.div`
 `;
 
 const Home: FC<HomeProps> = ({ faqList }) => {
-  const handleSubmit: FormEventHandler<HTMLFormElement> | undefined = (e) => {
-    e.preventDefault();
+  const handleSubmit: FormEventHandler<HTMLFormElement> | undefined = (
+    event,
+  ) => {
+    event.preventDefault();
     alert('Submitted');
```

Comment:
we need to replace alert... can you create task on that please?

Reply:
Created https://linear.app/lidofi/issue/UI-188/remove-alerts